schemas/oic.sec.didtype.json

```diff
@@ -16,18 +14,5 @@
           },
           "required": ["uuid"]
```

Comment:
@mkettrick if you look where my comment is applied you can see it's not highlighting the "required" part, but the "didtype" schema definition of "uuid":... where the "uuid" is defined in that case, it shows braces {} around the uuid data.  Anyway, I'm ignoring it if it does imply an extra set of braces.